api.planx.uk/types.ts

```diff
@@ -1,13 +1,19 @@
 import { PaymentRequest } from "@opensystemslab/planx-core/dist/types";
 import { GovUKPayment } from "@opensystemslab/planx-core/types";
 
+/**
+ * @deprecated Migrating to Node from planx-core
+ */
 export interface Node {
   id?: string;
   data?: Record<string, any>;
   edges?: Array<string>;
   type?: number;
 }
 
+/**
+ * @deprecated Migrating to Flow and FlowGraph from planx-core
+ */
```

Comment:
:+1: these are helpful reminders, thanks!

Reply:
Yep - hopefully it just means when we next need to reach for a type we avoid these ones 👌